packages/http/fetch/.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "rules": {
         "@typescript-eslint/no-empty-interface": "warn",
         "@typescript-eslint/ban-types": "off",
+        "no-unused-vars": "off",
```

Comment:
Is this interntional? 😅

Reply:
No it's not. I must have ctrl dot and selected the wrong option somewhere.
If you have a couple of minutes to revert that change and see where it complains, I'd appreciate it.

Reply:
No worries. Will take a look

Reply:
2 things here
- A number of request builders have model serializer/deserailizer imports that should have been dropped(doesn't look related to the latest changes) such as `import { createODataErrorFromDiscriminatorValue, deserializeIntoODataError, serializeODataError, type ODataError } from '../../../../models/oDataErrors/';`
- Correct syntax for  `// eslint-enable` is incorrect and should be /* eslint-enable */ for generated files.

I'll look into this and push updates to generation side to resolve these two things before updating the PR again.

Reply:
I have fixed the eslint comments in a separate PR.
As for this PR, I implemented some changes to avoid importing request/navigation metadata types when not necessary.
I still need to review the imports/usings for error and regular models.

Reply:
done, I left the error type import in anticipation of this
https://github.com/microsoft/kiota/issues/3811